umbraco/Umbraco.CMS.Backoffice
# Tiptap extension: Link/Unlink (URL Picker)

## Description

Adds the Link and Unlink toolbar buttons, using the URL Picker modal feature.

## Types of changes


## How to test?

Copy-paste links from TinyMCE RTE, select the link, press the toolbar button, check that the correct meta-data is loaded, e.g. URL, querystring, selected content/media.

Add text to the RTE, highlight text, select Link button, populate the URL Picker modal, submit. Select the link you create, press the toolbar button, check that the correct meta-data is loaded in the modal.
